### PR TITLE
multilabel_margin_loss implementation with pytorch frontend. 

### LINKED ISSUE #19764: multilabel_margin_loss

### PR DESCRIPTION
**Pull request by Huvinesh-Rajendran-12**
Close #19764

### CODE REVIEW

File: ivy/functional/frontends/torch/nn/functional/loss_functions.py [hunk 1]
```diff
@@ -613,3 +613,20 @@ def triplet_margin_with_distance_loss(
     loss = ivy.maximum(dist_pos - dist_neg + ivy.array(margin), ivy.array(0.0))
 
     return reduction(loss).astype(anchor.dtype)
+
+
+@to_ivy_arrays_and_back
+@with_unsupported_dtypes({"2.0.1 and below": ("float16", "bfloat16")}, "torch")
+def multilabel_margin_loss(input, target, reduction="mean"):
```
Comment (hello-fri-end) [inline ivy/functional/frontends/torch/nn/functional/loss_functions.py:620]: The function signature should match with the torch `multilabel_margin_loss` (https://pytorch.org/docs/stable/generated/torch.nn.functional.multilabel_margin_loss.html). `size_average` and `reduce` arguments are missing here.

**Review by hello-fri-end — CHANGES_REQUESTED** [review]
Hey @Huvinesh-Rajendran-12, thanks for creating this PR. Would you please address the comment I've left. Feel free to request another review when you are done with the changes. Thanks.

File: ivy/functional/frontends/torch/nn/functional/loss_functions.py (hunk 1 above)
Comment (Huvinesh-Rajendran-12) [reply ivy/functional/frontends/torch/nn/functional/loss_functions.py]: Hi @hello-fri-end , I have updated the args in my latest commit as with my implementation of multilabel_margin_loss. Thanks.

File: ivy_tests/test_ivy/test_frontends/test_torch/test_nn/test_functional/test_loss_functions.py [hunk 2]
```diff
@@ -984,3 +984,44 @@ def test_torch_triplet_margin_with_distance_loss(
...
+    ),
+    reduction=st.sampled_from(["none", "mean", "sum"]),
+    test_with_out=st.just(False),
+)
+def test_torch_multilabel_margin_loss(
+    *,
+    dtype_and_inputs,
+    reduction,
+    size_average,
+    reduce,
```
Comment (hello-fri-end) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_nn/test_functional/test_loss_functions.py:1009]: Where are you generating the value for these two?

File: ivy_tests/test_ivy/test_frontends/test_torch/test_nn/test_functional/test_loss_functions.py [hunk 3]
```diff
@@ -984,3 +984,44 @@ def test_torch_triplet_margin_with_distance_loss(
...
     )
+
+
+# multilabel_margin_loss
+
+
+@handle_frontend_test(
+    fn_tree="torch.nn.functional.multilabel_margin_loss",
+    dtype_and_inputs=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```
Comment (hello-fri-end) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_nn/test_functional/test_loss_functions.py:995]:
```suggestion
        available_dtypes=helpers.get_dtypes("valid"),
```
`valid` automatically returns all the datatypes supported by the function

File: ivy/functional/frontends/torch/nn/functional/loss_functions.py [hunk 4]
```diff
@@ -613,3 +613,23 @@ def triplet_margin_with_distance_loss(
     loss = ivy.maximum(dist_pos - dist_neg + ivy.array(margin), ivy.array(0.0))
 
     return reduction(loss).astype(anchor.dtype)
+
+
+@to_ivy_arrays_and_back
+@with_unsupported_dtypes({"2.0.1 and below": ("float16", "bfloat16")}, "torch")
+def multilabel_margin_loss(
+    input, target, reduction="mean", size_average=None, reduce=None
```
Comment (hello-fri-end) [inline ivy/functional/frontends/torch/nn/functional/loss_functions.py:621]: Sorry for nitpicking here. Could you please keep these in the same order as in the original pytorch function?https://pytorch.org/docs/stable/generated/torch.nn.MultiLabelMarginLoss.html#torch.nn.MultiLabelMarginLoss

**Review by hello-fri-end — CHANGES_REQUESTED** [review]

**Review by hello-fri-end — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**